### PR TITLE
Adding source_name to add_issue_select_details.

### LINKED ISSUE #330: NOT NULL constraint failed: Issues.source_name on add_issue_select_details
Pull request incoming.

### PR DESCRIPTION
**Pull request by MichaelFitzurka**
Fixes #330

### CODE REVIEW

**Comment by lordwelch** [comment]
The argument order is incorrect

**Comment by MichaelFitzurka** [comment]
The code works as written.
Can you be more specific as to the “incorrect” order?

File: comictaggerlib/comiccacher.py [hunk 1]
```diff
@@ -378,7 +378,13 @@ def get_volume_issues_info(self, volume_id: int, source_name: str) -> list[CVIss
         return results
 
     def add_issue_select_details(
-        self, issue_id: int, image_url: str, thumb_image_url: str, cover_date: str, site_detail_url: str
+        self,
+        source_name: str,
+        issue_id: int,
```
Comment (lordwelch) [inline comictaggerlib/comiccacher.py]: Here you specifically have source_name, and then issue_id

File: comictaggerlib/comicvinetalker.py [hunk 2]
```diff
@@ -677,7 +677,7 @@ def cache_issue_select_details(
         self, issue_id: int, image_url: str, thumb_url: str, cover_date: str, page_url: str
     ) -> None:
         cvc = ComicCacher()
-        cvc.add_issue_select_details(issue_id, image_url, thumb_url, cover_date, page_url)
+        cvc.add_issue_select_details(issue_id, self.source_name, image_url, thumb_url, cover_date, page_url)
```
Comment (lordwelch) [inline comictaggerlib/comicvinetalker.py:680]: Here you specifically have issue_id and then source_name

**Review by lordwelch — CHANGES_REQUESTED** [review]

File: comictaggerlib/comicvinetalker.py (hunk 2 above)
Comment (MichaelFitzurka) [reply comictaggerlib/comicvinetalker.py:680]: D'Oh! - It's that self in python's parameter list - throws off my count.

**Pull request merged**